### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by 22dt91**
I've created a new branch from scratch, now without trimming lines.
It contains my review for chap1->6

### CODE REVIEW

**Comment by Mte90** [comment]
Thanks a lot!
I will wait before to merge (and to review all the changes), I will need also your name to add in the thanks list at the end of the book :-)

I still don't know when I will do the 4th edition but maybe I can do just a minor release with those fix.

**Comment by 22dt91** [comment]
I've reviewed also the last chapters, I'm waiting for the new version (when you have time to review the PR).
Btw, I'm Denny Tremolada

File: manuscript/Book.txt
Comment (Mte90) [inline manuscript/Book.txt:1]: Can you revert this change and reput that file as it is needed to generated the pdf and the other versions?

**Review by Mte90 — CHANGES_REQUESTED** [review]
If you revert/put that file back I can merge your contribution :-)

File: manuscript/Book.txt
Comment (22dt91) [reply manuscript/Book.txt:1]: Done!

**Comment by Mte90** [comment]
Probably I will do a new release after the fosdem next february with all your fixes1
Thanks!

File: manuscript/appendix-3.md
Comment (DeinAlptraum) [inline manuscript/appendix-3.md:5]: Adding the `a` here is wrong, the sentence was correct before without it.

File: manuscript/appendix-3.md
Comment (DeinAlptraum) [inline manuscript/appendix-3.md:24]: `let` was correct here because we are talking about the mentor's past experiences.

File: manuscript/preface.md
Comment (DeinAlptraum) [inline manuscript/preface.md:17]: `maintainerz` is not a word, this should be `maintainers`

File: manuscript/conclusion.md
Comment (DeinAlptraum) [inline manuscript/conclusion.md:31]: `theme` is not a word, should be `there`

File: manuscript/conclusion.md
Comment (DeinAlptraum) [inline manuscript/conclusion.md:39]: This changes the meaning of the sentence.

File: manuscript/chapter4-gotta-catch-em-all.md
Comment (DeinAlptraum) [inline manuscript/chapter4-gotta-catch-em-all.md:171]: This was more correct with the `it` there

**Pull request merged**
